tasks/show_linters_issues/golangci_lint_parser.py

```diff
@@ -86,6 +92,22 @@ def filter_lints(lints_per_linter, filter_team: str = None, filter_linters: str
     return filtered_lints
 
 
+def count_lints_per_team(lints_per_linter, filter_linters: str = None):
+    """
+    Counts the lints owned by one team
+    """
+    list_filter_linters = filter_linters.split(',') if filter_linters else []
+    lints_count_per_team = defaultdict(lambda: defaultdict(lambda: 0))
+    for linter in lints_per_linter:
+        if linter in list_filter_linters:
+            for lint in lints_per_linter[linter]:
+                lint_owner = get_owner(lint[0])
+                lints_count_per_team[lint_owner][linter] += 1
+    for lint_owner in lints_count_per_team:
+        lints_count_per_team[lint_owner] = dict(lints_count_per_team[lint_owner])
```

Comment:
🐒 Nitpick :
If we format the printed string there's no need to convert `defaultdict` to `dict`